src/lib/converter/plugins/CommentPlugin.ts

```diff
@@ -144,7 +144,7 @@ export class CommentPlugin extends ConverterComponent
         const comment = reflection.parent.comment;
         if (comment) {
             let tag = comment.getTag('typeparam', reflection.name);
-            if (!tag) tag = comment.getTag('param', '<' + reflection.name + '>');
+            if (!tag) tag = comment.getTag('param', `<${reflection.name}>`);
```

Comment:
For consistency I personally use template literal anywhere there is interpolation